sys/shell/commands/sc_fib.c

```diff
@@ -71,7 +71,7 @@ int _fib_route_handler(int argc, char **argv)
     /* e.g. fibroute right now dont care about the adress/protocol family */
     if (argc == 1) {
         fib_print_routes();
-        return;
+        return 0;
```

Comment:
I think changes to this file are independent from your RPL refactorings and could be outsourced to its own PR. Reducing the diff slightly.